### PR TITLE
Add GitHub Actions for Jib CLI release

### PR DESCRIPTION
**Pull request by chanseokoh**
We need to upload an archive containing all the JAR files, but I think this is a good starting point.

### CODE REVIEW

**Comment by loosebazooka** [comment]
how does a release get triggered?

**Comment by chanseokoh** [comment]
It's manual. On the GitHub Actions page, there's a "Run workflow" button when selecting the "Release Jib CLI release" workflow.

image

**Comment by loosebazooka** [comment]
oh cool

File: .github/workflows/jib-cli-release.yml [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+
+      - name: Build Jib CLI
+        run: |
+          # TODO: run integration test? (Requries auth with GCP.)
+          ./gradlew clean build jib-cli:instDist
+
+      - name: Run Gradle release
+        run: |
+          git checkout -b cli_release_v${{ github.event.inputs.release_version }}
+          git config user.email "chanseok@google.com"
```
Comment (loosebazooka) [inline .github/workflows/jib-cli-release.yml:26]: I wonder if this can be configured to be the user triggering the event?

**Review by loosebazooka — APPROVED** [review]

File: .github/workflows/jib-cli-release.yml (hunk 1 above)
Comment (chanseokoh) [reply .github/workflows/jib-cli-release.yml:26]: Yeah, I need to figure this out eventually.

**Pull request merged**